website/components/blog/PostTitle.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import Link from "next/link"
+
+interface Props {
```

Comment:
This is totally fine and doesn't need any changes. What we could do though is to make React components independent from data structures of prismic, so that they are independent of backend solutions. I don't expect us to change prismic to anything else in the upcoming future, but if we were to change it, we would have to fit new api into the structure based on prismic.

Reply:
Not sure if I explained it clearly - ask on priv if you have questions

Reply:
Not needed, I understand that react component should just got props like: `title:string`, instead of receivin Prismic objects